### ISSUE TITLE
is “Instance Normalization” used in this paper?

### ISSUE THREAD

**Issue by Hsveh** [sniklaus/softmax-splatting#15]
Very impressive work! But I have some questions when I finished reading the paper. You mentioned that you apply "Instance Normalization" to ctxsyn in the paper "Context-aware Synthesis for Video Frame Interpolation", and I have 2 questions about the details.
(1) is “Instance Normalization” also used in this paper?
(2) if (1) is true, how do you jointly normalize the two input frames?
(2) if (1) is true, are the inputs of the feature extractor normalize?

**Comment by sniklaus** [comment]
Thank you for your kind words!

(1) Yes.
(2a) By computing the statistics jointly for the two images, instead of separately.
(2b) Yes.

**Comment by Fairydetail** [comment]
Hello，as the paper "Context-aware Synthesis for Video Frame Interpolation" described below, I have several questions.@sniklaus 
![image](https://user-images.githubusercontent.com/26640881/101303287-a5926d80-3878-11eb-8853-facdd2533467.png)

**Comment by sniklaus** [comment]
(1) Yes.
(2) No, we do not normalize the features used to compute the importance metric.
(3) We have trained our own PWC-Net with the instance normalization in place.
(4) Yes, it has to.
